### ISSUE TITLE
[BUG:CDVDGigahertz] keepAlive not working

### ISSUE THREAD

**Issue by knight-ryu12** [PCSX2/pcsx2#3455]
On using CDVD drive to boot PS2 game, CD drive does spin down, not archive anything at all by using keepAliveThread().

**Comment by MonJamp** [comment]
Could you expand on this? Does using a CD drive not work at all or is the drive not being opened properly?

**Comment by knight-ryu12** [comment]
neither. upon playing game on some CDVD drive, it get spin down constantly, creates spin-up time always.

**Comment by MonJamp** [comment]
As you're playing a game the disc spins slower and slower? Or do you mean it spins fast then slow then fast again?

**Comment by knight-ryu12** [comment]
disc spins during access, then some amount of time of no access results in disc spin down.

means that it need to spin up.

**Comment by MrCK1** [comment]
Closing since plugin no longer exists

**Comment by seta-san** [comment]
yeah. i think the ps2 keep a disc constantly sped up but windows pcs don't do that. pc games and media are more accommodating for variable disc access speeds. for the ps2 the cdvd speed needs to be kept as constant as anything else in the system like memory access speed.

**Comment by Mrlinkwii** [comment]
can this be checked on latest nightly ?
